tests/page-click.spec.ts

```diff
@@ -780,3 +780,29 @@ it('should click zero-sized input by label', async ({page}) => {
   await page.click('text=Click me');
   expect(await page.evaluate('window.__clicked')).toBe(true);
 });
+
+it('should block all events when hit target is wrong', async ({page, server}) => {
+  await page.goto(server.PREFIX + '/input/button.html');
+  await page.$eval('button', button => {
+    const blocker = document.createElement('div');
+    blocker.style.position = 'absolute';
+    blocker.style.width = '400px';
+    blocker.style.height = '400px';
+    blocker.style.left = '0';
+    blocker.style.top = '0';
+    document.body.appendChild(blocker);
+
+    const allEvents = [];
+    (window as any).allEvents = allEvents;
+    for (const name of ['mousedown', 'mouseup', 'click', 'dblclick', 'auxclick', 'contextmenu', 'pointerdown', 'pointerup'])
+      button.addEventListener(name, e => allEvents.push(e.type));
```

Comment:
Shouldn't this test verify that no events are fired on the blocker/window. How would events ever be fired on the button?

Reply:
Indeed. Changed as you suggest.